website/cue/reference/releases/0.19.0.cue

```diff
@@ -0,0 +1,569 @@
+package metadata
+
+releases: "0.19.0": {
+	date:     "2021-12-28"
+	codename: ""
+
+	whats_next: [
+		{
+			title: "Faster disk buffers"
+			description: """
+				We are in the process of replacing our current disk buffer
+				implementation, which leverages LevelDB in a way that doesn't
+				quite match common LevelDB use-cases, with a custom
+				implementaion specific to Vector's needs. The end result is
+				faster disk buffers.
+				"""
+		},
+		{
+			title:       "Component metric standardization"
+			description: """
+				We are in the process of ensuring that all Vector components
+				report a consistent set of metrics to make it easier to monitor
+				the performance of Vector.  These metrics are outlined in this
+				new [instrumentation
+				specification](\(urls.specs_instrumentation)).
+				"""
+		},
+	]
+
+	description: """
+		The Vector team is pleased to announce version 0.19.0!
+
+		We've been hard at work improving Vector's performance and were able to
+		move the needle 10-100% for most configurations in our [soak test
+		framework](https://github.com/vectordotdev/vector/tree/master/soaks/tests)
+		from the last release, `v0.18`.
+
+		Be sure to check out the [upgrade
+		guide](/highlights/2021-12-28-0-19-0-upgrade-guide.md) for breaking
+		changes in this release.
+		"""
+
+	changelog: [
+		{
+			type: "enhancement"
+			scopes: ["aws_s3 source", "delivery"]
+			description: """
+				Added end-to-end acknowledgement for the `aws_s3` source.
+				"""
+			pr_numbers: [10045]
+		},
+		{
+			type: "feat"
+			scopes: ["splunk_hec source", "splunk_hec sink"]
+			description: """
+				The `splunk_hec` source and sink have added support for the
+				acknowledgement part of Splunk's HTTP Event Collector protocol.
+				This improves delivery guarantees for data from Splunk clients
+				and when sending events to Splunk. See [the highlight
+				article](/highlights/2021-12-15-splunk-hec-improvements) for
+				more details.
+				"""
+			pr_numbers: [10444, 10162, 10135, 10350]
+		},
+		{
+			type: "enhancement"
+			scopes: ["vrl"]
+			description: """
+				VRL now allows for writing multi-line string literals by ending the line with a backslash (\\).
+
+				Example:
+
+				```text
+				.thing = "foo \\
+				bar"
+
+				assert(.thing == "foo bar")
+				```
+				"""
+			pr_numbers: [10149]
+		},
+		{
+			type: "enhancement"
+			scopes: ["influxdb_logs sink"]
+			description: """
```

Comment:
Do we want to shoutout the contributor here? I've always thought that was nice.

Reply:
👍 Good idea. I pushed 8f41411 which adds another cue field for this. We'll need to update the website to render it though. I'll leave that as a follow-up.